docs/_data/authors.yml

```diff
@@ -0,0 +1,15 @@
+stefan:
+  name: Stefan Arentz
+  image: https://avatars0.githubusercontent.com/u/28052?s=460&v=4
+  twitter: https://twitter.com/st3fan
+christian:
+  name: Christian Sadilek
+  image: https://avatars0.githubusercontent.com/u/472523?s=460&v=4
+  twitter: https://twitter.com/csadilek
+sebastian:
+  name: Sebastian Kaspari
+  image: https://avatars0.githubusercontent.com/u/89638?s=88&v=4
+  twitter: https://twitter.com/Anti_Hype
+fernando:
+  name: Fernando García Álvarez
+  image: https://avatars3.githubusercontent.com/u/4409856?s=400&v=4
```

Comment:
@fercarcedo Is this okay? Do you have a Twitter handle you want to use? (It's optional)